src/bors/handlers/retry.rs

```diff
@@ -0,0 +1,95 @@
+use std::sync::Arc;
+
+use crate::PgDbClient;
+use crate::bors::handlers::{PullRequestData, deny_request, has_permission};
+use crate::bors::merge_queue::MergeQueueSender;
+use crate::bors::{Comment, RepositoryState};
+use crate::github::{GithubUser, PullRequestNumber};
+use crate::permissions::PermissionType;
+
+pub(super) async fn command_retry(
+    repo_state: Arc<RepositoryState>,
+    db: Arc<PgDbClient>,
+    pr: PullRequestData<'_>,
+    author: &GithubUser,
+    merge_queue_tx: &MergeQueueSender,
+) -> anyhow::Result<()> {
+    if !has_permission(&repo_state, author, pr, PermissionType::Try).await? {
+        deny_request(&repo_state, pr.number(), author, PermissionType::Try).await?;
+        return Ok(());
+    }
+
+    let pr_model = pr.db;
+
+    if pr_model.stalled() {
```

Comment:
```suggestion
    if pr_model.is_stalled() {
```
To keep consistency with `is_approved` and similar methods.